Detections/ASimDNS/imDns_IPEntity_DnsEvents.yaml

```diff
@@ -35,7 +35,7 @@ query: |
   | extend TI_ipEntity = coalesce(NetworkIP, NetworkDestinationIP, NetworkSourceIP,EmailSourceIpAddress,"")
   | summarize LatestIndicatorTime = arg_max(TimeGenerated, *) by IndicatorId);
```

Comment:
We have a new PR that states that the test for active should follow the summarize. It makes sense.